scipy/signal/filter_design.py

```diff
@@ -3653,6 +3653,232 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, rp=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    A notch filter is a band-stop filter with a narrow bandwidth
+    (high quality factor). It rejects a narrow frequency band and
+    leaves the rest of the spectrum little changed.
+
+    Parameters
+    ----------
+    w0 : float
+        Normalized frequency to remove from a signal. It is a
+        scalar that must satisfy  0 < w0 < 1, with w0 = 1 corresponding
+        to half of the sampling frequency.
+    bw : float
+        Normalized filter bandwidth (the diference between up and lower
+        passband edges).
+    rp : float, optional
```

Comment:
That is a good explanation. I think you are correct that the stopband in our case is a single point (I didn't think about it this way). So yes, let's keep `rp` then.

About the definition of `rp` (former `ab`). I thought of it more like the strength of the filter's attenuation (is it partially correct thinking?), but now I realize that it can (and probably should) be thought as passband sort of ripple (or fluctuations). So your definition makes more sense to me now, I think it's good.

To further clarify for myself I'm going to plot amplitude responses with different `rp` values.